contrib/lua5.4-luafilesystem/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "lua5.4-luafilesystem"
+pkgver = "1.8.0"
+pkgrel = 0
+makedepends = ["lua5.4-devel"]
+pkgdesc = "File system library for lua"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://lunarmodules.github.io/luafilesystem"
+source = f"https://github.com/lunarmodules/luafilesystem/archive/refs/tags/v{pkgver.replace('.','_')}.tar.gz"
+sha256 = "16d17c788b8093f2047325343f5e9b74cccb1ea96001e45914a58bbae8932495"
+
+
+def init_configure(self):
+    self.tool_flags["CFLAGS"] += [
+        f"-I{self.profile().sysroot / 'usr/include/lua5.4'}",
+        "-shared",
+        "-fPIC",
+    ]
+
+
+def do_build(self):
+    from cbuild.util import compiler
+
+    cc = compiler.C(self)
+    cc.invoke(["src/lfs.c"], "lfs.so")
```

Comment:
you can drop the `init_configure` and pass the args as `flags=` and `ldflags=` respectively here

Reply:
just the regular makefile stuff: overriding cflags, relying on gnu install behavior, gratuitoulsy using gmake only syntax, and at times broken lua search paths

Reply:
i think manually specifying the flags here is less likely to cause headaches and patches down the line